modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -1162,6 +1162,19 @@ void ONNXImporter::handleNode(const opencv_onnx::NodeProto& node_proto_)
                     layerParams.type = "Scale";
                 }
             }
+            else if (!haveVariables)
+            {
+                Mat inp0 = getBlob(node_proto, 0);
+                Mat inp1 = getBlob(node_proto, 1);
+                if (inp0.size != inp1.size && inp1.total() != 1)
+                    CV_Error(Error::StsNotImplemented, "Constant multiply with different shapes");
+
+                Mat out = isDiv ? inp0 / inp1 : inp0.mul(inp1);
+                out = out.reshape(1, inp0.dims, inp0.size);
```

Comment:
Do we need to reshape inputs instead? (to avoid failure of operation itself due to strong checks: `inp0.size != inp1.size` is allowed with `.total() == 1`)

Reply:
Or we can use something like that:
```cpp
if (isDiv)
  divide(1.0, inp1, inp1);
Mat out = inp0.mul(inp1);
```